### DISCUSSION TITLE
Discover new device added to Ring

### DISCUSSION THREAD

**Discussion by cherry919** [tsightler/ring-mqtt#487] [Support Q&A]
Hello,
Thank you for making this add on.  I'm using Ring as my Z-Wave controller, this add on is very helpful.  I've added a new Z-Wave device to Ring.  How do I refresh MQTT to pickup the new device so it appears in my Overview?

**Comment by tsightler** [comment]
Just restart the addon, that's all that's needed, assuming it's a supported device.
